### PR TITLE
add topics for 2020.2 release notes

### PR DESCRIPTION
**Pull request by lbalicki**
I added the structure for the 2020.2 release notes with some of the most important PRs to the release_notes.rst file.

### CODE REVIEW

**Comment by HenKlei** [comment]
I think #1110 and #1205 should be our highlights if they make it into the release.

**Comment by renefritze** [comment]
Does @pymor/all object to moving all but the current release notes into a separate .rst document that is included into `release_notes.rst? 

I found searching through the cumbersome with the current length, which IMO has no upside for us since it's processed into a single output by sphinx either way.

**Comment by pmli** [comment]
Thanks, @lbalicki. I've edited sections I'm more familiar with. I also merged the sections about inner products and norms since it seems they are related.

Maybe it's fine to move the "VectorArrays are iterable" part to "Further notable improvements", but I'm not sure.

**Comment by pmli** [comment]
> Does @pymor/all object to moving all but the current release notes into a separate .rst document that is included into `release_notes.rst?
> 
> I found searching through the current format cumbersome with the current length, which IMO has no upside for us since it's processed into a single output by sphinx either way.

I guess then it makes sense to have release notes for all releases in different files, such that it's not necessary to move release notes from current to old for every future release.

**Comment by renefritze** [comment]
> > Does @pymor/all object to moving all but the current release notes into a separate .rst document that is included into `release_notes.rst?
> > I found searching through the current format cumbersome with the current length, which IMO has no upside for us since it's processed into a single output by sphinx either way.
> 
> I guess then it makes sense to have release notes for all releases in different files, such that it's not necessary to move release notes from current to old for every future release.

Right. That's better.

**Comment by HenKlei** [comment]
Since I am not allowed to push directly into this branch, I created a new PR for my part of the release notes, see #1207.

**Comment by renefritze** [comment]
> > > Does @pymor/all object to moving all but the current release notes into a separate .rst document that is included into `release_notes.rst?
> > > I found searching through the current format cumbersome with the current length, which IMO has no upside for us since it's processed into a single output by sphinx either way.
> > 
> > 
> > I guess then it makes sense to have release notes for all releases in different files, such that it's not necessary to move release notes from current to old for every future release.
> 
> Right. That's better.

Have a look: https://github.com/pymor/pymor/compare/releasenotes_2020_2...release_not_refactor

**Comment by renefritze** [comment]
Can I get a thumbs up reaction of everyone who still intends to add to this PR?

**Comment by pmli** [comment]
> Can I get a thumbs up reaction of everyone who still intends to add to this PR?

I'm not sure I can fill in the remaining sections, but I can try helping with editing...

**Comment by renefritze** [comment]
Rebasing on master so @TiKeil can rebase on this to get CI in his PR #1216 working again.

File: docs/source/release_notes/2020.2.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+A new tutorial on :doc:`using pyMOR for accelerating the solution of linear PDE-constrained
+optimization problems <../tutorial_optimization>` has been added `[#1205] <https://github.com/pymor/pymor/pull/1205>`_.
+The tutorial showcases the new features added in `[#1110] <https://github.com/pymor/pymor/pull/1110>`_
+and also discusses general questions on using model order reduction for this class
+of optimization problems.
+
+The tutorial :doc:`'Projecting a Model' <../tutorial_projection>` explains how pyMOR builds an
+online-efficient reduced order model via (Petrov-)Galerkin projection onto a given reduced space
+`[#1084] <https://github.com/pymor/pymor/pull/1084>`_.
+Apart from a mathematical introduction, the user is introduced to some core elements
```
Comment (HenKlei) [inline docs/source/release_notes/2020.2.rst:51]:
```suggestion
Apart from a mathematical introduction, the user is introduced to some core elements of
```

File: docs/source/release_notes/2020.2.rst [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+pyMOR's internal architecture that realize the projection.
+
+A tutorial on :doc:`linear time-invariant systems <../tutorial_lti_systems>` was added
+and the existing :doc:`balanced truncation tutorial <../tutorial_bt>` was appropriately simplified
+`[#1141] <https://github.com/pymor/pymor/pull/1141>`_.
+
+All tutorials now include a 'launch binder' button which allows to directly 
+run the tutorial code in the web browser `[#1181] <https://github.com/pymor/pymor/pull/1181>`_.
+
+In order to consolidate our documentation all remaining Jupyter notebooks from the `notebooks/`
```
Comment (HenKlei) [inline docs/source/release_notes/2020.2.rst:61]:
```suggestion
In order to consolidate our documentation, all remaining Jupyter notebooks from the `notebooks/`
```

File: docs/source/release_notes/2020.2.rst [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+Backward incompatible changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Updated Model interface
+~~~~~~~~~~~~~~~~~~~~~~~
+To make the simultaneous computation of multiple |Model| output quantities such internal state,
+output, or error estimates more efficient better customizable a :meth:`~pymor.models.interface.Model.compute`
+method was added to the |Model| interface which is now responsible for the computation of all
+relevant data that can be gathered from the simulation of a |Model|
+`[#1113] <https://github.com/pymor/pymor/pull/1113>`_.
```
Comment (HenKlei) [inline docs/source/release_notes/2020.2.rst:117]: Looks like this sentence got mixed up a bit.

File: docs/source/release_notes/2020.2.rst [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+:meth:`~pymor.models.interface.Model.compute`.
+Existing custom |Models| have to be adapted to the new architecture.
+
+The `estimate` method has been renamed to :meth:`~pymor.model.interface.Model.estimate_error`
+`[#1041] <https://github.com/pymor/pymor/pull/1041>`_.
+The old method is deprecated and will be removed in the next release.
+
+Further, to simplify interoperability with third-party packages,
+the model outputs, i.e., the results of :meth:`~pymor.models.interface.output`,
+are not longer generic |VectorArrays|, but NumPy arrays.
```
Comment (HenKlei) [inline docs/source/release_notes/2020.2.rst:129]:
```suggestion
are no longer generic |VectorArrays|, but NumPy arrays.
```

**Comment by TiKeil** [comment]
needs to be rebased on master before #1219 and #1220 have a chance to pass the tests.

**Pull request merged**